public/serviceworker.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable array-callback-return */
+const CACHE_NAME = "version-1";
+const urlsToCache = [ 'index.html', 'offline.html' ];
+const self = this;
+/* eslint-disable array-callback-return */
+self.addEventListener('install', (event) => {
+    event.waitUntil(
+        caches.open(CACHE_NAME)
+            .then((cache) => {
+                console.log('Opened cache');
```

Comment:
if you are debugging using these logs, print them only if DEBUG is enabled. see my comment on DEBUG somewhere above in the review